BetterHolidays/markets/holidays.py

```
@@ -0,0 +1,159 @@
+            return None
+      
+        if self.end and day > self.end:
+            return None
+        
+        if type(self.type) == Holiday:
+            return self.type(
+                date=day,
+                name=self.name,
+            )
+                
+        elif type(self.type) == PartialTradingDay:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)